packages/grafana-ui/src/components/Card/CardContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { HTMLAttributes, ReactNode } from 'react';
+import { css, cx } from '@emotion/css';
+import { GrafanaThemeV2 } from '@grafana/data';
+import { PopoverContent, Tooltip } from '../Tooltip/Tooltip';
+import { styleMixins, stylesFactory, useTheme2 } from '../../themes';
+
+/**
+ * @public
+ */
+export interface ContainerProps extends HTMLAttributes<HTMLOrSVGElement> {
+  /** Content for the card's tooltip */
+  tooltip?: PopoverContent;
+  /** Disable pointer events for the Card, e.g. click events */
+  disableEvents?: boolean;
+  /** No style change on hover */
+  disableHover?: boolean;
+  /** Custom container styles */
+  className?: string;
+}
+
+const Container = ({ children, tooltip, disableEvents, disableHover, className, ...props }: ContainerProps) => {
+  const theme = useTheme2();
+  const { container } = getCardContainerStyles(theme, disableEvents, disableHover);
+
+  return tooltip ? (
+    <Tooltip placement="top" content={tooltip} theme="info">
+      <div {...props} className={cx(container, className)}>
+        {children}
+      </div>
+    </Tooltip>
+  ) : (
+    <div {...props} className={cx(container, className)}>
+      {children}
+    </div>
+  );
```

Comment:
feels strange that the whole card has a toolttip?

Reply:
That's according to the designs, there are cases when we need to display tooltips on card hover, also it only applies when the card has a `tooltip` prop.

Reply:
ok, feels very strange to have a tooltip on something so large

Reply:
Where is this tooltip prop used?

Reply:
Cannot find this used anywhere, can we remove it?

Reply:
Card can also be a list component with just a title and description. In that case having a tooltip makes sense there. It's just our Card component is trying to handle too many use cases imo.

Reply:
But if that comes up the user can just wrap it in a Tooltip component, just a big object like wide or high card a tooltip seems like a bad idea 90% of the time

Reply:
Just realised myself that users can just wrap the card in tooltip if they need it, good point, I'll remove it.